lua/nvim-navic/init.lua

```diff
@@ -287,6 +286,11 @@ function M.get_location(opts, bufnr)
 	return ret
 end
 
+function M.get_location(opts, bufnr)
+	local data = M.get_data(bufnr)
+    return M.format_data(opts, data)
```

Comment:
LGTM! Just fix some inconsistent indentations, use tabs instead of spaces.

Reply:
Turns out `indent-blankline.nvim` interferes with my `listchars` so I didn't even notice tabs were used in this project! Fixed in 0f2732e695af43b6fc07e5bd0cde66b4a54dfee1.